tests/testapp/models.py

```diff
@@ -1 +0,0 @@
-"""Empty demo app our tests can assign models to."""
```

Comment:
is this file still in the repo? its not very clear from the diff.

Reply:
I moved it to `django_tables/tests/testapp/models.py` and this docstring eventually got removed.

I actually moved `tests` into `django_tables` which I was hoping wouldn't mess with the diff too much.